apps/quick-learn-backend/src/routes/lesson/lesson.service.ts

```diff
@@ -8,12 +8,16 @@ import { en } from '@src/lang/en';
 import { UserTypeIdEnum } from '@quick-learn/shared';
 import { PaginationDto } from '../users/dto';
 import { PaginatedResult } from '@src/common/interfaces';
+import { DataSource, Repository } from 'typeorm';
 
 @Injectable()
 export class LessonService extends PaginationService<LessonEntity> {
   constructor(
     @InjectRepository(LessonEntity) repo,
     private courseService: CourseService,
+    @InjectRepository(LessonEntity)
```

Comment:
The pagination service exposes a repository which is the same as here injecting it